### PR TITLE
Move packages-next to packages

### PR DESCRIPTION
**Pull request by JedWatson**
Been meaning to do this since we got rid of the last legacy packages and removed the original `packages` directory.

This moves `packages-next` to `packages` and updates all the references.

I also found a few things we missed (old config in `package.json` etc) for things that have been removed, and cleaned it up.

### CODE REVIEW

**Review by timleslie — CHANGES_REQUESTED** [review]
Requires a changeset

**Comment by JedWatson** [comment]
@timleslie

> Requires a changeset

What were you expecting this to include?

File: .changeset/early-dingos-retire.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+"@keystone-next/website": patch
+"@keystone-next/example-auth": patch
+"@keystone-next/example-custom-field": patch
+"@keystone-next/admin-ui-utils": patch
+"@keystone-next/auth": patch
+---
+
+Move packages-next to packages
```
Comment (timleslie) [inline .changeset/early-dingos-retire.md:9]:
```suggestion
Move source code from the `packages-next` to the `packages` directory.
```

File: .changeset/early-dingos-retire.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+"@keystone-next/website": patch
+"@keystone-next/example-auth": patch
+"@keystone-next/example-custom-field": patch
+"@keystone-next/admin-ui-utils": patch
+"@keystone-next/auth": patch
+---
```
Comment (timleslie) [inline .changeset/early-dingos-retire.md:7]:
```suggestion
"@keystone-next/cloudinary": patch
"@keystone-next/document-renderer": patch
"@keystone-next/fields": patch
"@keystone-next/fields-document": patch
"@keystone-next/keystone": patch
"@keystone-next/session-store-redis": patch
"@keystone-next/testing": patch
"@keystone-next/types": patch
"@keystone-next/utils": patch
"@keystone-next/api-tests-legacy": patch
---
```

Need to ship patch releases for all the packages which have changed their package.json files.

**Review by timleslie — APPROVED** [review]
👍

**Pull request merged**